theme/ItaliaTheme/Blocks/_info.scss

```diff
@@ -35,32 +40,31 @@ $info-block-tertiary-font-color-with-bg: $black;
     }
 
     .bg-alert-primary {
-      color: $info-block-primary-font-color;
-      border-color: $info-block-primary-color;
+      color: var(--info-block-primary-font-color);
+      border-color: var(--info-block-primary-color);
       .draftjs-buttons a {
-        background-color: $info-block-primary-color;
+        background-color: var(--info-block-primary-color);
         &:hover {
-          background-color: darken($info-block-primary-color, 10%);
+          background-color: calc(var(--info-block-primary-color) - 10%);
```

Comment:
Non funziona così per rendere più scuro un colore. Per farlo puoi usare i colori in formato HSL oppure i filtri css. https://stackoverflow.com/questions/1625681/dynamically-change-color-to-lighter-or-darker-by-percentage-css